src/domain/app/footer/Footer.tsx

```diff
@@ -133,13 +150,7 @@ const Footer: React.FC = () => {
             copyrightHolder={t('footer.copyrightHolder')}
             copyrightText={t('footer.copyrightText')}
             backToTopLabel={t('footer.backToTopLabel')}
-            logo={
-              <Logo
-                src={logoLanguage === 'sv' ? logoSv : logoFi}
-                size="medium"
-                alt={t('appName')}
-              />
-            }
+            logo={<Logo src={logo} size="medium" alt={t('appName')} />}
```

Comment:
This is not related to this PR, but maybe footer logo should have same alt text as the logo in header has:
```
logo={<Logo src={logoSrcFromLanguage(locale)} alt={t('logo')} />}
```
so 
```
logo={<Logo src={logo} size="medium" alt={t('logo')} />}
```

Reply:
Good point, might as well fix it right away: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/269/commits/ca90eec015eb582b616fb04cf848d058df3ddf92